src/components/RocketInfo.js

```diff
@@ -0,0 +1,43 @@
+/* eslint camelcase: 0 */
```

Comment:
Please use camelcase in your codebase. camelcase make your code not clean.

Reply:
Hello @aliabbani 
This camercase error disabler is because of the format of the data returned from the API, which I have no control over.

Reply:
Hi @ReshyResh, 
That s understandable. thank you for explaining.